js/popup.js

```diff
@@ -73,33 +73,24 @@ var logout = function() {
     $("#add-bug").addClass("disabled");
 }
 
-function addGithub(url) {
+function addGithub(url, tabTitle) {
     var path = url.pathname.split('/');
     var bugNum = path[path.length - 1];
     var bugOwner = path[1];
     var bugRepo = path[2];
-    var type = 'Unkown'
+    var type = 'Unkown';
     if (url.pathname.indexOf('issues') > -1) {
         type = 'Issue';
     }
     else if (url.pathname.indexOf('pull') > -1) {
         type = 'Pull';
     }
-    var bugJson = $.ajax({
-        type: "Get",
-        url: url,
-        dataType: "html",
-        success: function (data) {
-            var prefix = bugRepo + ": #" + bugNum
-            var body = $(data).filter('meta[name="description"]').attr("content");
```

Comment:
I notice that `'meta[name="description"]'`now seems to contain the repo's description instead of the issue's like it used to. That now seem to be in `'meta[property="og:description"]'` It would be better to use that than just set it as an empty string like you do below.